pipeline.py

```diff
@@ -9,6 +9,8 @@
     CreatePVC,
     DeletePVC,
     mount_pvc,
+    set_image_pull_policy,
```

Comment:
Is `set_image_pull_policy` every used?

Reply:
i left in there so when we do need it all we have to do is uncomment below (for now, while we zero in on what's required in the ilab1.2+ image)